testing/tests/DevExpress.viz.sparklines/bulletTooltip.tests.js

```diff
@@ -157,43 +203,71 @@ QUnit.test('dxBullet get TooltipFormatObject', function(assert) {
     }, {}]);
 });
 
-QUnit.test('Default Tooltip text', function(assert) {
+QUnit.test('Default template should be used when customizeTooltip is nota defined', function(assert) {
```

Comment:
plz,  check everywhere
```suggestion
QUnit.test('Default template should be used when customizeTooltip is not specified', function(assert) {
```